src/components/LayerCollection/LayerPOLine.js

```diff
@@ -57,32 +57,30 @@ class LayerPOLine extends Component {
     const { lineMutator, onCancel } = this.props;
 
     try {
-      const response = await lineMutator.POST(newLine);
-      if (!response.id) throw new Error(response);
+      await lineMutator.POST(newLine);
       onCancel();
     } catch (e) {
       const errors = await e.json();
```

Comment:
instead of `errors` you get here JSON representation of response. So `response` might be a better name.
https://developer.mozilla.org/en-US/docs/Web/API/Body/json